modules/vaos/app/services/ccra/configuration.rb

```diff
@@ -40,13 +40,16 @@ def mock_enabled?
     def connection
       Faraday.new(api_url, headers: base_request_headers, request: request_options) do |conn|
         conn.use :breakers
+        conn.request :camelcase
```

Comment:
Note: To match EPS. Note these affect how the token data is received (e.g. string vs symbol, "access_token" vs :access_token)